5.6/test/run

```diff
@@ -362,7 +363,7 @@ function run_tests() {
   create_container $name $envs
   test_connection "$name" "$USER" "$PASS"
   echo "  Testing scl usage"
-  test_scl_usage $name 'mysql --version' '5.6'
+  test_scl_usage $name 'mysql --version' '5.7'
```

Comment:
Same as for image name.

`test_scl_usage $name 'mysql --version' "${VERSION}"`